### ISSUE TITLE
Using with Aurelia templates - not working

### ISSUE THREAD

**Issue by telbiyski** [cheeriojs/cheerio#1039]
When you try to parse Aurelia Templates, they always have <template> tag.
Cheerio put this tag in the <head> instead in the <body>

**Comment by stevenvachon** [comment]
I cannot make sense of this. Please try again

**Comment by telbiyski** [comment]
github remove my html tags from the comment (dull)
when you load aurelia template as `. load('<template>......</template>')`
and then `console.log($.html())`
you have `<html><head><template>...........</template></head><body></body></html>`
and nothing is working, because the body is empty.
If i remove `<template>` tag - everything is working fine, without Aurelia.
`<template>` tag is required for Aurelia.

**Comment by stevenvachon** [comment]
@telbiyski what happens when you open that template in a web browser? Same thing, I'd imagine. This is the new default behaviour for cheerio.

**Comment by telbiyski** [comment]
if you can't fix it, i have to use the old version 0.22.0, but i think you have to make some fix for this. maybe option to wrap the content in `<html><head><body>....` or not, like: `.load(html, false)` to work as the old cheerio.

**Comment by stevenvachon** [comment]
Duplicate of https://github.com/cheeriojs/cheerio/issues/1024

**Comment by thomas-darling** [comment]
This caught me by surprise too, so I did some digging, and here's what's going on:

```js
const $ = cheerio.load("test");
console.log($.html());
```
This logged `test` in `0.22.0`, now it logs `<html><head></head><body>test</body></html>`.

```js
const $ = cheerio.load("<div>test</div>");
console.log($.html());
```
This logged `test` in `0.22.0`, now it logs `<html><head></head><body><div>test</div></body></html>`.

```js
const $ = cheerio.load("<template>test</template>");
console.log($.html());
```
This logged `test` in `0.22.0`, now it logs `<html><head><template>test</template></head><body></body></html>` which makes no sense.

Now, comparing this to Google Chrome, this is indeed aligned with how a browser handles those cases.

That being said though, it makes it _much_ harder to use this library for it's intended purpose - how am I supposed to reliably find the node containing the content `test`?

Also, this actually breaks the example at the very top of the cheerio `readme.md`:

```js
const cheerio = require('cheerio')
const $ = cheerio.load('<h2 class="title">Hello world</h2>')

$('h2.title').text('Hello there!')
$('h2').addClass('welcome')

$.html()
//=> <h2 class="title welcome">Hello there!</h2>
```

In my opinion, this change undermines the entire premise of what this library is supposed to do, breaking a whole lot of dependent code as a consequence.

Also, just last month, the `readme.md` was updated in a commit titled `Clarify project scope in README file` - to quote the commit message for that:

> The project frequently receives bug reports from new users who expect
full web browser behavior. The project documentation does not make it
particularly clear that such functionality is out-of-scope for the
project.

And the actual change introduced in that commit included this:

> Cheerio parses markup and provides an API for traversing/manipulating the resulting data structure. It does not interpret the result as a web browser does.

So in conclusion, this change seems very, very wrong.

@stevenvachon Can you explain how I am supposed to parse a string that may be _either_ `<template>test</template>` or `<div>test</div>`, and then reliably get the content of that element as a string - i.e. `test`?

**Comment by stevenvachon** [comment]
```js
const $ = cheerio.load("");
const body = $("body");

body.html("<template>test</template>");

console.log(body.html());
//-> <template>test</template>
```

**Comment by telbiyski** [comment]
ok, this is fine.
thanks
